pkg/validate.go

```diff
@@ -0,0 +1,20 @@
+package pkg
+
+import "go.buf.build/protocolbuffers/go/prometheus/prometheus"
+
+const (
+	clusterIDLabelName string = "cluster_id"
+)
+
+// ValidateClusterIDs returns a list of cluster IDs found in the request
+func ValidateClusterIDs(req *prometheus.WriteRequest) map[string]int {
```

Comment:
The function name is not entirely correct, since it doesn't validate, just return the different cluster ids. How about `findClusterIds`?

Reply:
thanks @pb82! Will update this